deployment/docker-compose.latest.yml

```diff
@@ -0,0 +1,89 @@
+# Development Setup.
+# Uses the latest commit of the `master` branch to run each service.
+# Start individual services using the following command::
+#
+#   docker-compose up <service_1> <service_2> [..]
+#
+# Add the `-d` flag to daemonize the container.
+#
+# Updating the images is as easy as this:
+#
+#   docker-compose pull && docker-compose restart
+#
+# ..Note::
+#
+#   The above commands assume you're already in the directory where
+#   where this docker-compose file is located.
+version: '3.4'
+
+x-defaults: &defaults
+  restart: "no"
```

Comment:
Why is this `"no"`

Reply:
There was confusion on my side - this file should the `local` dev file. I'll submit a fix.